netbox/netbox/views/generic/object_views.py

```diff
@@ -90,6 +91,7 @@ class ObjectChildrenView(ObjectView, ActionsMixin, TableMixin):
     child_model = None
     table = None
     filterset = None
+    template_name = 'inc/tab_view.html'
```

Comment:
Yes, the idea of the issue raised was to avoid writing such code to help reduce the boilerplate code. I think it would be best to remove this need in v3.6